core/ide/che-core-ide-stacks/src/main/resources/predefined-stacks.json

```diff
@@ -0,0 +1,710 @@
+[
```

Comment:
why these stacks are part of ide core module ? as it's more linked to the core/platform than IDE